frontend/src/components/HandbookOverlay/HandbookOverlay.css

```diff
@@ -1,55 +1,36 @@
 .handbook-overlay {
 	display: flex;
-	flex-direction: row;
+	flex-direction: column;
 	height: 85vh;
-	background-color: var(--handbook-background-colour);
-	color: var(--handbook-text-colour);
 	aspect-ratio: 7/8;
 }
 
-.handbook-overlay .close-button {
-	position: absolute;
-	top: 0.25rem;
-	right: 0.25rem;
-}
-
-.handbook-overlay .close-button:hover svg rect {
-	fill: black;
-	stroke: white;
+.handbook-overlay header {
```

Comment:
Oh, now I can guess why you changed the other header into a div! This is quite a loose css selector, so all header elements within the overlay would've received these styles.

In this case, it makes more sense to use the classname as selector:
`.handbook-overlay .overlay-header`